src/utils/prices.ts

```diff
@@ -29,7 +29,8 @@ export function computeRealizedLPFeePercent(
     )
   } else {
     percent = ONE_HUNDRED_PERCENT.subtract(
-      trade.route.pools.reduce<Percent>(
+      // TODO(judo): compute across all swaps
+      trade.swaps[0].route.pools.reduce<Percent>(
```

Comment:
So i _think_ we just do this per route then sum.. something like:

```
const totalPercent = new Percent(0, 0)
for (const swap of swaps) {
    const overallPercent = <compute using amount and total amount>
    overallPercent.subtract(... // same logic as before)
    totalPercent.add(overallPercent)
}
```

Looks like Moody wrote this though so should check with him